exec/exec.go

```diff
@@ -105,18 +107,39 @@ func Output(ctx context.Context, command ...string) ([]byte, []byte, error) {
 	}
 
 	cmd := exec.CommandContext(ctx, command[0], arg...)
-
-	var outbuf, errbuf bytes.Buffer
 	cmd.Stdout = &outbuf
 	cmd.Stderr = &errbuf
 
 	if err := cmd.Start(); err != nil {
-		return nil, nil, errors.Wrapf(err, "unable to run command %s", cmd)
+		return nil, nil, 0, errors.Wrapf(err, "unable to run command %s", cmd)
 	}
 
-	if err := cmd.Wait(); err != nil {
-		return outbuf.Bytes(), errbuf.Bytes(), err
+	err = cmd.Wait()
```

Comment:
well, we do not ignore it. The err in this case might contain important information:
1. The real error (os/exec was unable to find the script for instance)
2. The script was executed successfully but the program exited with non zero code. (in this case the error will be not nil, but it's not a real error, it's just a way to give the user exit code)

the next function is trying to figure out if the error was real or it was just the exit code != 0. The later option means do not fail and continue execution.